data/socials.js

```diff
@@ -15,3 +15,23 @@ export const socialMediaLinks = [
     iconClass: "fab fa-behance",
   },
 ];
+
+export const socialMediaSvgLinks = [
```

Comment:
Instead of using SVG, we should utilize the iconClass from font-awesome and define it in `socialMediaLinks` array.

Reply:
changes made, We needed to upgrade our local Font Awesome files to version 6.6.0 in order to display the Upwork icon.